parametric/test_span_sampling.py

```diff
@@ -234,31 +274,42 @@ def test_keep_span_with_stats_computation_sss010(test_agent, test_client: _TestT
 
 
 @pytest.mark.skip_library("dotnet", "Not implemented")
-@pytest.mark.skip_library("golang", "Not implemented")
+@pytest.mark.skip_library("golang", "The Go tracer does not have an easy way to modulate trace sampling once started")
```

Comment:
no easy way, or no way at all?

Reply:
No way that I'm aware of or that anyone from the Go team offered.

Reply:
maybe worth saying that there's no known way then, else it sounds like we're just lazy